pandas/core/indexes/base.py

```diff
@@ -681,7 +681,44 @@ def _values(self):
         return self.values
 
     def get_values(self):
-        """ return the underlying data as an ndarray """
+        """
+        Return `Index` data as an `ndarray`.
+
+        Because this is a wrapper function around Index.values, it is useful
```

Comment:
Hmm, I suspect that the real reason for this is compatability with `Series.get_values`, which does more work (sparse transformation).

Reply:
regardless. I would personally just omit this section.

Reply:
The validator requires an extended summary, so I would write about the reason you mention, but I do not find any information about 'sparse transformation'. Could you point me to a term/link on which I can follow up?